### PR TITLE
Feature map

### PR DESCRIPTION
**Pull request by JohannDallmann**

### CODE REVIEW

File: backend/src/main/java/de/neuefische/backend/security/SecurityConfig.java [hunk 1]
```diff
@@ -42,6 +43,8 @@ public SecurityFilterChain filterChain(HttpSecurity http) throws Exception {
                             ("/user/login"), ("/user/register")).permitAll();
                     auth.requestMatchers(
                             ("/user/**")).authenticated();
+                    auth.requestMatchers(
+                            ("/locations/**")).authenticated();
                     auth.anyRequest().permitAll();
```
Comment (MJaschewski) [inline backend/src/main/java/de/neuefische/backend/security/SecurityConfig.java]: anyRequest should never be permitAll. Let be denyAll and you know when didn't apply authentication properly

File: frontend/package.json [hunk 2]
```diff
@@ -42,5 +45,8 @@
       "last 1 safari version"
     ]
   },
-  "proxy": "http://localhost:8080"
+  "proxy": "http://localhost:8080",
+  "devDependencies": {
+    "@types/leaflet": "^1.9.3"
```
Comment (MJaschewski) [inline frontend/package.json]: dont use devDependencies. Add it to normal dependencies

**Review by MJaschewski — CHANGES_REQUESTED** [review]
Looks nice! Some changes would be recommened.

**Comment by JohannDallmann** [comment]
changes added. ready to continue

**Review by MJaschewski — APPROVED** [review]

**Pull request merged**